tiledb/sm/serialization/array.cc

```diff
@@ -358,6 +365,257 @@ Status array_open_from_capnp(
   return Status::Ok();
 }
 
+void fragments_timestamps_to_capnp(
+    uint64_t start_timestamp,
+    uint64_t end_timestamp,
+    capnp::ArrayFragmentsTimestamps::Builder* array_fragments_builder) {
+  array_fragments_builder->setStartTimestamp(start_timestamp);
+  array_fragments_builder->setEndTimestamp(end_timestamp);
+}
+
+void fragments_timestamps_from_capnp(
+    const capnp::ArrayFragmentsTimestamps::Reader& array_fragments_reader,
+    uint64_t& start_timestamp,
+    uint64_t& end_timestamp) {
+  start_timestamp = array_fragments_reader.getStartTimestamp();
+  end_timestamp = array_fragments_reader.getEndTimestamp();
+
+  if (start_timestamp > end_timestamp) {
+    throw std::logic_error(
+        "[fragments_timestamps_from_capnp] Deserialized timestamps are "
+        "invalid");
+  }
```

Comment:
As per our style guide, let's return here `{start_timestamp, end_timestamp}` so that it's more evident that those are return values.